dev.slimevr.SlimeVR.yml

```diff
@@ -0,0 +1,90 @@
+app-id: dev.slimevr.SlimeVR
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '42'
+separate-locales: false
+command: slimevr
+finish-args:
+  # Network access
+  - --share=network
+  # Needed for graphics
+  - --share=ipc
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Add JRE bin on path
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --env=GDK_BACKEND=wayland
```

Comment:
so what happen if you only have X11?

Reply:
It should run, doesn't GTK fallback to X11?

Reply:
it doesnt run in wayland by default for some reason if I dont put that

Reply:
now it should work